packages/dart_frog_cli/lib/src/dev_server_runner/dev_server_runner.dart

```diff
@@ -59,9 +69,7 @@ class DevServerRunner {
     @visibleForTesting io.ProcessSignal? sigint,
     @visibleForTesting ProcessStart? startProcess,
     @visibleForTesting ProcessRun? runProcess,
-    @visibleForTesting Exit? exit,
```

Comment:
Before this PR, the whole CLI process was killed if there was any compilation error of the dev server code. With thins such as the daemon, that cannot happen.  The removal of this parameter illustrates that change.